packages/itwinui-react/src/core/utils/components/InputWithIcon.tsx

```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { polymorphic } from '../index.js';
+
+export const InputWithIcon = polymorphic.div('iui-input-with-icon');
+InputWithIcon.displayName = 'InputWithIcon';
```

Comment:
Is it necessary to provide a `displayName` when the name is the same as the component name?

Reply:
Yes, component name is usually not enough especially when using arrow functions and/or `forwardRef`


React devtools will show "ForwardRefComponent" for the following snippet:
```js
const Stuff = React.forwardRef((props, ref) => {});
```